Resources/Prototypes/Entities/Mobs/NPCs/prawn.yml

```diff
@@ -0,0 +1,270 @@
+- type: entity
+  name: Prawn
+  id: MobPrawn
+  parent: SimpleMobBase
```

Comment:
this should probably inherit from `BaseMobOrganic`, as that'd avoid most of the copy and pasted boilerplate here.

Reply:
might be easier if i remove the pull request. Then work on it more. How do i remove it?